src/main/java/com/glodblock/github/common/parts/PartFluidStorageBus.java

```diff
@@ -291,38 +285,7 @@ public void getBoxes(final IPartCollisionHelper bch) {
 
     @Override
     public void onNeighborChanged() {
-        TileEntity tile = this.getTile();
```

Comment:
- [ ] Could you explain why this logic is redundant now?
 - [ ] Could you also check classic dupe scenarios using this bus with neighbours?
 - [ ] Could also check system with two subnets which are connected using itemstorage and fluidstorage buses in bidirectional way. Does it produce duplication of fluid drops in item terminal (it may be blinking situation)
 - [ ] Could you check setup with separate subnet with only cable and fstorage bus and fiber. Does it not continue to work after dismount fiber using wrench?

Reply:
> * [ ]  Could you explain why this logic is redundant now?
I'll try but I think it's been redundant for a long time. The current behavior is that if the Part or TE opposite to the FStorage Bus is a PartFluidInterface, a IFluidHandler or null we do an immediate full reset.

1. case PartFluidInterface: In this case the FStorage Bus is still kissing the Fluid Interface so nothing actually changed. Any network changes that happen because of a neighbor update are handled through `postChange` and not by doing a full reset.
2. case IFluidHandler: This is one I'm not totally sure about since there are too many possibilities. I believe an immediate full reset is not necessary since fluid handlers are handled as monitors and will be updated each tick anyway.
3. case null: In the case of an interface we'll get changes posted because of `com.glodblock.github.common.parts.PartFluidInterface#gridChanged` and in the case of a IFluidHandler it will be updated next tick.

> * [ ]  Could you also check classic dupe scenarios using this bus with neighbours?
I don't know any classic dupe scenarios.

> * [ ]  Could also check system with two subnets which are connected using itemstorage and fluidstorage buses in bidirectional way. Does it produce duplication of fluid drops in item terminal (it may be blinking situation)
Fluid drops are not duplicated if only 1 network has a discretizer. They are duplicated if both networks have a discretizer, this has nothing to do with this PR however.

> * [ ]  Could you check setup with separate subnet with only cable and fstorage bus and fiber. Does it not continue to work after dismount fiber using wrench?
The FStorage Bus doesn't work when the fiber is disconnected.